src/EVM.hs

```diff
@@ -176,6 +176,7 @@ unknownContract :: Expr EAddr -> Contract
 unknownContract addr = Contract
   { code        = UnknownCode addr
   , storage     = AbstractStore addr Nothing
+  , t_storage   = AbstractStore addr Nothing
```

Comment:
I think we prefer not using underscore, so maybe tStorage or tempStorage would work better?